### PR TITLE
Create CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by KaiSzuttor**

### CODE REVIEW

**Comment by hmenke** [comment]
Why do we need this?  Did some [SJW](https://en.wikipedia.org/wiki/Social_justice_warrior) from administration complain?

**Comment by KaiSzuttor** [comment]
No its just informative, see https://opensource.guide/code-of-conduct/

**Comment by fweik** [comment]
CoC is recommended by GitHub foe FOSS projects.

**Comment by KaiSzuttor** [comment]
It's great, it even increases coverage!

**Comment by fweik** [comment]
Don't use my personal email. Maybe we should have some functional email @espressomd.org ?

**Comment by RudolfWeeber** [comment]
On Wed, Dec 06, 2017 at 09:46:56AM +0000, Kai Szuttor wrote:
> It's great, it even increases coverage!
Let's talk about this at the next ESPResSo meeting

**Comment by jonaslandsgesell** [comment]
What was the outcome?

**Comment by KaiSzuttor** [comment]
https://github.com/espressomd/espresso/pull/3445

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+[INSERT CONTACT METHOD].
```
Comment (KaiSzuttor) [inline CODE_OF_CONDUCT.md:64]: who will be responsible?

**Pull request closed without merging**